packages/core/core/src/AssetGraph.js

```diff
@@ -103,12 +103,14 @@ export default class AssetGraph extends ContentGraph<AssetGraphNode> {
   onNodeRemoved: ?(nodeId: NodeId) => mixed;
   hash: ?string;
   envCache: Map<string, Environment>;
+  #unsafeToIncrementallyBundle: boolean = false;
 
   constructor(opts: ?SerializedAssetGraph) {
     if (opts) {
       let {hash, ...rest} = opts;
       super(rest);
       this.hash = hash;
+      this.#unsafeToIncrementallyBundle = false;
```

Comment:
Just to clarify for myself and other reviewers, here are the conditions that need to be met for incremental bundling: non-production mode, incremental flag on, must have previous assetgraph, some entry(ies) have changed, but not added (or altered in ways that warrant a rebundle). 

I think I'm not missing anything ? @joeyslater